pkg/operator/hive/hive_controller.go

```diff
@@ -200,29 +210,40 @@ var _ reconcile.Reconciler = &ReconcileHiveConfig{}
 // ReconcileHiveConfig reconciles a Hive object
 type ReconcileHiveConfig struct {
 	client.Client
-	scheme                *runtime.Scheme
-	kubeClient            kubernetes.Interface
-	apiextClient          *apiextclientv1beta1.ApiextensionsV1beta1Client
-	apiregClient          *apiregclientv1.ApiregistrationV1Client
-	discoveryClient       discovery.DiscoveryInterface
-	dynamicClient         dynamic.Interface
-	restConfig            *rest.Config
-	hiveImage             string
-	hiveOperatorNamespace string
-	hiveImagePullPolicy   corev1.PullPolicy
-	syncAggregatorCA      bool
-	managedConfigCMLister corev1listers.ConfigMapLister
+	scheme                            *runtime.Scheme
+	kubeClient                        kubernetes.Interface
+	apiextClient                      *apiextclientv1beta1.ApiextensionsV1beta1Client
+	apiregClient                      *apiregclientv1.ApiregistrationV1Client
+	discoveryClient                   discovery.DiscoveryInterface
+	dynamicClient                     dynamic.Interface
+	restConfig                        *rest.Config
+	hiveImage                         string
+	hiveOperatorNamespace             string
+	hiveImagePullPolicy               corev1.PullPolicy
+	syncAggregatorCA                  bool
+	managedConfigCMLister             corev1listers.ConfigMapLister
+	ctrlr                             controller.Controller
+	servingCertSecretWatchEstablished bool
+	mgr                               manager.Manager
 }
 
 // Reconcile reads that state of the cluster for a Hive object and makes changes based on the state read
 // and what is in the Hive.Spec
 func (r *ReconcileHiveConfig) Reconcile(request reconcile.Request) (reconcile.Result, error) {
-
 	hLog := log.WithField("controller", "hive")
 	hLog.Info("Reconciling Hive components")
 
 	// Fetch the Hive instance
 	instance := &hivev1.HiveConfig{}
+
+	// We only support one HiveConfig per cluster, and it must be called "hive". This prevents installing
+	// Hive more than once in the cluster.
+	if request.NamespacedName.Name != hiveConfigName {
+		hLog.WithField("hiveConfig", request.NamespacedName.Name).Warn(
```

Comment:
We can leave as is, but you do not need to include `NamespacedName`.

```suggestion
	if request.Name != hiveConfigName {
		hLog.WithField("hiveConfig", request.Name).Warn(
```